src/metabase/events.clj

```diff
@@ -138,10 +138,6 @@
     (let [model (topic->model topic)]
       (get object (keyword (format "%s_id" model))))))
 
-(def ^{:arglists '([object])} object->user-id
-  "Determine the appropriate `user_id` (if possible) for a given `object`."
-  (some-fn :actor_id :user_id :creator_id))
```

Comment:
Removed this since we're now using the value bound in `api/*current-user-id*` by default